randovania/gui/preset_settings/item_pool_tab.py

```diff
@@ -230,13 +230,26 @@ def _on_update_maximum_starting(self, value: int):
 
     def _create_categories_boxes(self, pickup_database: PickupDatabase, size_policy):
         self._boxes_for_category = {}
+        all_categories = list(pickup_database.pickup_categories.values())
 
         categories = set()
         for standard_pickup in pickup_database.standard_pickups.values():
             if not standard_pickup.hide_from_gui:
                 categories.add(standard_pickup.pickup_category)
 
-        all_categories = list(pickup_database.pickup_categories.values())
+        # FIXME: this really shouldn't be done this way. This is just temporary to have ammo in their own category.
+        broad_to_category = {
+            "beam_related": "beam",
+            "morph_ball_related": "morph_ball",
+            "missile_related": "missile",
+        }
+        for ammo_pickup in pickup_database.ammo_pickups.values():
+            if ammo_pickup.broad_category.name in broad_to_category:
+                proper_name = broad_to_category[ammo_pickup.broad_category.name]
+                categories.add(next(filter(lambda p: p.name == proper_name, all_categories)))
+            else:
+                categories.add(ammo_pickup.broad_category)
```

Comment:
Is it possible to generalise it? Searching for broad_categories which are not referenced by any main item / `standard_pickups`?

Reply:
Not that I know of. The issue currently is that `broad_category` *is* nowhere referenced for most ammo. I.e. for missiles, all the main items refer to it as `missile`.

Reply:
I won't check it in its entirety on mobile but https://github.com/randovania/randovania/blob/cb87d71c37d1f789be933381fae97baaf162f3ff/randovania/games/am2r/pickup_database/pickup-database.json#L85-L99
at least here is definitely the broad_category present. If it's true in general, you could generate list of all broad categories from the ammo pickups and subtract all broad categories you found in the standard pickups.
Could maybe combine it into the already existing iteration, so it is not that expensive and it would remove the game specific junk.